pages/tools/git-signing.md

```diff
@@ -0,0 +1,225 @@
+
+## Backup your private key in Google Drive
+
+If you are not using a password manager, you can copy the keys to Google Drive
+as a backup. However, you need to also keep the files on your computer. If you
+are using a password manager, be sure to periodically backup your password
+database to [Google Drive]({% page "tools/google-drive/" %}). This will
+automatically include your signing keys.
+
+## Setup your keys for use
+
+### Using KeePassXC
+
+#### Setup KeePassXC to manage your SSH keys
+
+Open KeePassXC and open the settings (`⌘+,` on macOS, or the settings are
+locatedunder Tools -> SSH Agent in Keepass for Windows) and click the button on
+the left sidebar that says "SSH Agent." Ensure the "Enable SSH Agent
+integration" checkbox is checked, then click the OK button at the bottom.
+
+#### Add your keys to KeePassXC
+
+If you already have an entry in KeePassXC for your GitHub account, double-click
+it to open it in editing mode. Otherwise, create a new entry. On the left side
+of the window, click the "Advanced" button. On the bottom half of the new
+screen, there is a section titled "Attachments." Click the "Add" button on the
+right side, and use the file browser to find the key you created above. Look for
+the files called `git-signing-key` and `git-signing-key.pub`, and add them both.
+
+Once you have added the files to KeePassXC, you can delete them from your
+desktop. You will only be using them from KeePassXC from here onwards.
+
+#### Tell KeePassXC that these are keys
+
+Click the button that says "SSH Agent" on the left sidebar. Look for the section
+labeled "Private key" that includes a drop-down box labeled "Attachment." Click
+in the drop-down box and select the private key you added as an attachment.
+(**IMPORTANT:** Use the _private_ key, the one that is just `git-signing-key`,
+not the public key!). The sections below should then populate with information
+such as "Fingerprint," "Comment," and "Public key." Click the OK button at the
+bottom to save your entry.
+
+Finally, right-click on your entry and click the option near the bottom of the
+pop-up menu that says "Add key to SSH Agent."
+
+### Setup your keys manually
+
+#### macOS
+
+Open a terminal window and enter the following command:
+
+```sh
+ssh-add --apple-use-keychain ~/.ssh/git-signing-key
+```
+
+You will be prompted to enter the password you created for your key. Using the
+`--apple-use-keychain` option will store your password securely in the macOS
+keychain where it will only be available when you are logged into your computer.
+
+#### Linux
+
+Open a terminal window and enter the following command:
+
+```sh
+ssh-add ~/.ssh/git-signing-key
+```
+
+Note that you may be required to enter the password associated with your key
+every time you create a new git commit.
+
+## Configure git to sign commits
+
+By default, git uses a format called GPG for commit signing. However, GPG keys
+are more cumbersome to create and manage, so we recommend SSH keys, which is
```

Comment:
Thought: if this is meant to be generally accessible beyond TTS (ie, the general public can view our Handbooks), "cumbersome" may be difficult for someone who is not a native English speaker. It's a good word, though.